spipe/main.c

```diff
@@ -18,14 +18,20 @@
 
 #include "pushbits.h"
 
+struct accept_state {
```

Comment:
1) I was trying to keep this as similar to https://github.com/Tarsnap/spiped/commit/49470ac389c187a69311fe024efa9f549b5cfe9b as possible.

2) I figured that you'd rather see

```
static int
callback_conndied(void * cookie)
{
        struct accept_state * A = cookie;
```
instead of
```
static int
callback_conndied(void * cookie)
{
        struct int * conndone = cookie;
```

3) although spiped has an intermediate layer between `int main` and the `callback_conndone` (namely `dispatch.c`), spipe doesn't.  So if there's going to be any `void * cookie`, we need to prepare that inside `int main`.